modules/vaos/app/services/vaos/jwt.rb

```diff
@@ -17,6 +17,10 @@ def token
       ::JWT.encode(payload, rsa_private, 'RS512')
     end
 
+    def self.decode(jwt, key = nil, verify = true, options = {})
+      ::JWT.decode(jwt, key, verify, options)
+    end
```

Comment:
Does this need to exist? Can't `vaos_logging#decode_jwt` call `JWT.decode(jwt, key, verify, options)` directly?

Reply:
we're not verifying the signature. You'll notice the actual call to the decode method used in specs and the code has the verify flag as false. This is because the JWT is signed using staging certs that are unavailable locally and in test environment. It's mostly there for show, and shouldn't be causing performance problems -- but it's handy for verifying on a staging instance.

Reply:
I'm not seeing how any of this is relevant to the initial question 😬 .

Reply:
To be clear, I'm asking why we cant delete the entire `self.decode(jwt, key = nil, verify = true, options = {})` class method from this `VAOS::JWT` class. It seems like it's only usage is in `VAOS::Middleware::VaosLogging` (which could be [updated to use `::JWT` directly](https://github.com/department-of-veterans-affairs/vets-api/pull/3871/files#r382709855))

Don't mean to belabor such an unimportant question, just trying to make sure we're on the same page.